client/src/App.js

```diff
@@ -1,21 +1,41 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { addPlayer, startGame } from './actions/hanabiActions'
+import Game from './components/Game'
 
 class App extends Component {
```

Comment:
this is now a stateless component as well, since we are using Redux. It might become a class at a later instance, but we have a useless constructor  which signifies we should use a pure component for now 

```javascript
const App = (props) => {
  return (
    <div className="App">
      <Game
        startGame={props.actions.startGame}
        addPlayer={props.actions.addPlayer}
        game={props.game}
        />
    </div>
  );
}
```